src/sass/components/_table.scss

```diff
@@ -0,0 +1,38 @@
+.table-wrapper {
+	overflow-x: auto;
+	width: 100%;
+	margin: 0 auto;
+}
+
+.c-table {
```

Comment:
It would be better to add some more specific name to this element because we probably do not want to have the same table for the whole application. For example .c-table-afffixes-schedule